src/api/ucc_status.h

```diff
@@ -31,6 +31,8 @@ typedef enum {
     UCC_ERR_INVALID_PARAM               =   -3,
     UCC_ERR_NO_MEMORY                   =   -4,
     UCC_ERR_NO_RESOURCE                 =   -5,
+    /* General purpose return code without specific error */
+    UCC_ERR_NO_MESSAGE                  =   -6,
```

Comment:
Minor : Can you make this comment doxygen friendly? Example: https://github.com/openucx/ucc/blob/b790f356fef88981f196e828bd1ca1de5025947d/src/api/ucc.h#L982

Reply:
Done